### ISSUE TITLE
BanHammer

### ISSUE THREAD

**Issue by ghost** [grandwazir/BanHammer#128]
ik this isnt really a issue but is it possible to hand over this project.
i would like to use this plugin and might wanna update this with my friends for 1.12.2

**Comment by grandwazir** [comment]
Sure you can just fork and do what you want with it; the code is open
source. You may need to build BukkitUtilities as well let me know if you
need some help with that.

**Comment by ghost** [comment]
What version did this use btw

**Comment by ghost** [comment]
there is 1 issue. i can't find old buildtools for 1.7.10

**Comment by grandwazir** [comment]
Buildtools? Which dependency exactly?

**Comment by grandwazir** [comment]
The details should be in pom.xml - it specifies all the versions.

**Comment by ghost** [comment]
well in order to build stuff thats needed for BanHammer i need to compile
https://github.com/grandwazir/bukkit-utilities-commands/blob/master/pom.xml

but as far as i know there isnt a version for it anymore (buildtools currently only have 1.8 - 1.13.2)

and this needs 1.7.9

**Comment by grandwazir** [comment]
Just compile it against 1.8 and see what happens. Any breakages should be
really minor which you can fix.

**Comment by Thrasilias** [comment]
hey, today i was trying to get this compiled and failed since there is no maven-parent repo

**Comment by grandwazir** [comment]
Yes, the repo has been down for a while. You can however download a copy from the download page on GitHub.

-------- Original Message --------
